src/components/Sidebar.tsx

```diff
@@ -116,6 +117,7 @@ export const Sidebar: React.FunctionComponent<SidebarProps> = () => {
       <div className="flex flex-col">
         <SettingsModal />
       </div>
+      <ReactTooltip effect='solid' backgroundColor='#374151' arrowColor='#374151' className="text-xs"  />
```

Comment:
So if I wanna add tooltip in another part of Studio, e.g. in Editor, then I should copy that component and use `data-tip` attributes? 🤔  I think that component that wraps given component and add tooltip will be better, something like:

```tsx
<Tooltip content={...}>
  //...some component
</Tooltip>
```

is there any another tooltip component in the OpenSource?